test/editable_components/editable_content/properties_test.js

```diff
@@ -0,0 +1,41 @@
+require('../../setup');
+
+describe('EditableContent', function() {
+  const helpers = require('./helpers');
+  const c = helpers.constants;
```

Comment:
Couldn't see this get used again in the file? Is it needed?

Reply:
Fixed this now - there were a few of these. I was juts copying and pasting  the top of the file each time!